### PR TITLE
Add group edit

### PR DESCRIPTION
**Pull request by itkg-nanne**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2045
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/335
https://github.com/open-orchestra/open-orchestra-user-bundle/pull/138
https://github.com/open-orchestra/open-orchestra-base-bundle/pull/122
https://github.com/open-orchestra/open-orchestra-model-interface/pull/271
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/710

### CODE REVIEW

File: GroupBundle/Form/Type/GroupRoleType.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+use Symfony\Component\Form\FormView;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Class GroupRoleType
+ */
+class GroupRoleType extends AbstractType
+{
+
```
Comment (alavieille) [inline GroupBundle/Form/Type/GroupRoleType.php]: remove this line

File: GroupBundle/Form/Type/GroupRoleType.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    /**
+     * Returns the name of this type.
+     *
+     * @return string The name of this type
+     */
+    public function getName()
+    {
+        return 'oo_group_role';
+    }
+
```
Comment (alavieille) [inline GroupBundle/Form/Type/GroupRoleType.php]: remove this line

File: Backoffice/GeneratePerimeter/Strategy/NodeGeneratePerimeterStrategy.php [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    /**
+     * format perimeter
+     *
+     * @param array $treeNodes
+     *
+     * @return array
+     */
+    protected function formatPerimeter(&$treeNodes)
+    {
+        $path = $treeNodes['node']['path'];
```
Comment (itkg-nanne) [inline Backoffice/GeneratePerimeter/Strategy/NodeGeneratePerimeterStrategy.php]: The result come from repository 'findTreeNode'. So, if there is a 0 index, there is a 'node' index. On my point of view, i should just test 'path'?

**Comment by alavieille** [comment]
:+1:

**Pull request merged**